src/pages/about/index.tsx

```diff
@@ -1,13 +1,55 @@
 import Button from 'components/button'
 import Content from 'components/content'
-import Layout from 'components/layout'
-import { Component, createElement, ReactNode } from 'react'
+import DonateABox from 'components/donateabox'
+import { createElement, FunctionComponent } from 'react'
+import { Link } from 'react-router-dom'
 import styles from './style.scss'
 
-export default class About extends Component {
-    public render(): ReactNode {
-        return <Content>
-            <h2>About</h2>
-        </Content>
-    }
+const About: FunctionComponent = () => {
+    return <Content className={styles.about}>
+        <h2><span className='light'>About donateABLE</span></h2>
+        <div className={styles.items}>
+           <section>
+               <DonateABox char='WHO' fontSize='60'/>
```

Comment:
![image](https://user-images.githubusercontent.com/18175917/78184263-2de18880-7437-11ea-9e02-da04a9fbf49c.png)

The 'who' is overflowing the box, I had only ever expected people would use a single character